nextflow.config

```diff
@@ -106,7 +106,7 @@ profiles {
        }
       }
     } else if (params.illumina) {
-        process.container = "docker://genomicmedicinesweden/gms-artic-illumina:latest"
+        process.container = "docker://genomicmedicinesweden/artic-ncov2019-illumina:latest"
```

Comment:
Do not merge until this line is fixed. It should be `process.container = "docker://genomicmedicinesweden/gms-artic-illumina:latest"` (Updated 13 hours ago)? The one you link now was latest updated 6 months ago.

The container you link now was latest updated

Reply:
This is now fixed in this PR.